UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API.UnitTests/Services/EssWebhookServiceTests.cs

```diff
@@ -165,18 +231,29 @@ private BatchDetail GetBatchDetail()
             };
         }
 
-        private FssSearchResponseCache GetResponseCache()
+        private FssSearchResponseCache GetS63CacheResponse()
+        {
+            return new FssSearchResponseCache()
+            {
+                BatchId = "7b4cdf10-adfa-4ed6-b2fe-d1543d8b7272",
+                PartitionKey = "DE416050",
+                RowKey = "2|0|ADDS",
+                Response = JsonConvert.SerializeObject(GetBatchDetail())
+            };
+        }
```

Comment:
Accept RowKey all details are same in GetS57CacheResponse() method can we change GetS63CacheResponse() to GetS63CacheResponse(string exchangeSetStandard) and set RowKey = $("2|0|{0}", exchangeSetStandard) to avoid duplicate code please.